### PR TITLE
feat(@vtmn/svelte): add price component

### PR DESCRIPTION
**Pull request by Tlahey**
Creation of the component VtmnPrice.
The component will accept as props:
- size (default value: `medium`)
- variant (default value: `default`)

Todo list : 
- [x] Create a `VtmnPrice.svelte` component
- [x] Add the input definitions of the component
- [x] Add the component in the exports of the `package.json`
- [x] Integrate the component in the rollup
- [x] Create a storybook for the component

### CODE REVIEW

File: packages/sources/svelte/package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "./actions/VtmnButton/VtmnButton.svelte": "./dist/VtmnButton.js",
     "./actions/VtmnLink/VtmnLink.svelte": "./dist/VtmnLink.js",
     "./forms/VtmnTextInput/VtmnTextInput.svelte": "./dist/VtmnTextInput.js",
-    "./overlays/VtmnPopover/VtmnPopover.svelte": "./dist/VtmnPopover.js"
+    "./overlays/VtmnPopover/VtmnPopover.svelte": "./dist/VtmnPopover.js",
+    "./indicators/VtmnPrice/VtmnPrice.svelte": "./dist/VtmnPrice.js"
```
Comment (lauthieb) [inline packages/sources/svelte/package.json:69]: For alphabetical order :)
```suggestion
    "./indicators/VtmnPrice/VtmnPrice.svelte": "./dist/VtmnPrice.js",
    "./overlays/VtmnPopover/VtmnPopover.svelte": "./dist/VtmnPopover.js"
```

File: packages/sources/svelte/src/components/indicators/VtmnPrice/VtmnPrice.svelte [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+   * @defaultValue undefined
+   */
+  export let id;
+
+  /**
+   * Label apply on the component
+   * @type {string}
+   * @defaultValue undefined
+   */
+  export let label;
```
Comment (lauthieb) [inline packages/sources/svelte/src/components/indicators/VtmnPrice/VtmnPrice.svelte:17]: As the label will be inside the `<slot />`, is it possible to delete and set your `aria-label` manually when you use the components by spreading `$$restProps` as you we did in other Svelte components?
Thanks!

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thanks @Tlahey, just small changes :)

**Review by GaspardMathon — CHANGES_REQUESTED** [review]
I made some suggestions, I don't know a lot about svelte so don't hesitate to make a feedback  👍

**Review by lauthieb — APPROVED** [review]
Thanks a lot!!!!

**Review by lauthieb — APPROVED** [review]

**Pull request merged**